ContinueReadingWidget/WMFTodayContinueReadingWidgetViewController.swift

```diff
@@ -86,19 +86,10 @@ class WMFTodayContinueReadingWidgetViewController: UIViewController, NCWidgetPro
     }
     
     func hasNewData() -> Bool{
-        
-        guard let session = SessionSingleton.sharedInstance() else {
-            return false
-        }
-        
-        guard let historyEntry = session.dataStore.historyList.mostRecentEntry() else {
+        guard let openArticleURL = UserDefaults.wmf_userDefaults().wmf_openArticleURL() else {
```

Comment:
Ultimately, I'd like to move this out of user defaults and into a `WMFKeyValue`. However, since this property exists in `userDefaults` already, it's fine to keep for now.